### PR TITLE
Fixed issue #17347: 'Class Twig_Environment not found' when uploading an image

### PR DESCRIPTION
**Pull request by gabrieljenik**
Using autloader from composer

### CODE REVIEW

**Comment by olleharstedt** [comment]
Reverted the merge, sorry.

File: third_party/yiiext/twig-renderer/ETwigViewRenderer.php [hunk 1]
```diff
@@ -67,8 +67,8 @@ class ETwigViewRenderer extends CApplicationComponent implements IViewRenderer
 
     function init()
     {
-        require_once Yii::getPathOfAlias($this->twigPathAlias).'/Autoloader.php';
-        Yii::registerAutoloader(array('Twig_Autoloader', 'autoload'), true);
+        //require_once Yii::getPathOfAlias($this->twigPathAlias).'/Autoloader.php';
```
Comment (olleharstedt) [inline third_party/yiiext/twig-renderer/ETwigViewRenderer.php:70]: Why did you have to change this?

Comment (olleharstedt) [reply third_party/yiiext/twig-renderer/ETwigViewRenderer.php:70]: Long-term goal is to rename third_party to vendor, as per composer convention, and then include non of it in the repository. Instead, `composer install` should be part of release script. That means we won't be able to touch any of those files. We're not there yet, however, but for yiiext, "composer update" will probably remove any change you did to it.

**Comment by gabrieljenik** [comment]
New version: https://github.com/LimeSurvey/LimeSurvey/pull/1913

**Pull request merged**